examples/extensions/commands/custom_commands.rst

```diff
@@ -0,0 +1,174 @@
+.. _examples_extensions_custom_commands:
+
+
+Custom commands
```

Comment:
This should be in the `reference/commands/custom_commands` section.
It would be better if this has a reference style, describing the all the decorators, the arguments passed, etc. but in a neutral way, like in the rest of the reference.